src/test/groovy/com/lesfurets/jenkins/TestRunClosure.groovy

```diff
@@ -0,0 +1,47 @@
+package com.lesfurets.jenkins
+
+import com.lesfurets.jenkins.unit.BasePipelineTest
+import org.junit.Before
+import org.junit.Test
+
+import static com.lesfurets.jenkins.unit.MethodCall.callArgsToString
+import static org.junit.Assert.assertTrue
+
+class TestRunClosure extends BasePipelineTest {
+
+    @Override
+    @Before
+    void setUp() throws Exception {
+        super.setUp()
+    }
+
+    @Test
+    void should_execute_without_errors() throws Exception {
```

Comment:
Just an UX question: Do you want to test an exactly closure?
Or you would like to test an "inline" script 
```
runScript("""
   echo Hello World!
""")
```

Reply:
My use case is to test a part of code called in a big pipeline, that's why I did it with a closure.
But an inline script may also be interesting, I will give it a try!